runner/tasks.py

```diff
@@ -95,6 +96,9 @@ def process_triggers():
 @shared_task
 def create_run_task(run_id, inputs, output_directory=None):
     logger.info("Creating and validating Run")
+    if settings.DUMP_JSON:
```

Comment:
I don't like putting this in a task. Prints like this for debug are fine, but I think we should remove them from production code. I don't like the idea of generating files in which are not used anywere.

Reply:
This is required for development, debug, and manual run setup as per https://github.com/mskcc/beagle/issues/162 and https://github.com/mskcc/beagle/issues/150. Otherwise there is no straightforward method to actually see what your Job JSON and final output CWL look like. Its easier to leave it in as a feature than to have to re-add it every single time you need to use it for debugging.

Reply:
This is required for development, debug, and manual run setup as per https://github.com/mskcc/beagle/issues/162 and https://github.com/mskcc/beagle/issues/150. Otherwise there is no straightforward method to actually see what your Job JSON and final output CWL look like. Its easier to leave it in as a feature than to have to re-add it every single time you need to use it for debugging. There are probably better ways to implement this but it will take a lot more time to refactor and build.